tests/unit/test_parse_args.py

```diff
@@ -39,6 +39,7 @@ def expected_args(**kwargs):
         "start_date": None,
         "strict": False,
         "tags": False,
+        "template": None,
```

Comment:
It's odd that the tests fail when this value is se to `False`. I didn't expect that.